src/steps/email-links-validation.ts

```diff
@@ -55,14 +56,24 @@ export class EmailLinksValidationStep extends BaseStep implements StepInterface
         ]);
       }
 
+      const storageUrl: string = inbox.items.reverse()[position - 1].storage.url;
```

Comment:
- [ ] This may need to be moved further down after some sanity checks.  I'm getting the following:

![Screen Shot 2020-03-03 at 6 12 38 PM](https://user-images.githubusercontent.com/3496491/75837741-a393ff00-5d7a-11ea-849d-341fc41a96d8.png)